src/wp-includes/meta.php

```diff
@@ -1379,6 +1379,7 @@ function sanitize_meta( $meta_key, $meta_value, $object_type, $object_subtype =
  *                                         the meta key will be registered on the entire object type. Default empty.
  *     @type string     $type              The type of data associated with this meta key.
  *                                         Valid values are 'string', 'boolean', 'integer', 'number', 'array', and 'object'.
+ *     @type string     $label             A human-readable label of the data attached to this meta key.
```

Comment:
We should also add a new `@since` tag to the `register_meta` mentioning the change.

Example: ``@since 6.7.0 Added the `label` argument.``

Reply:
You're totally right. I just added it in this commit: https://github.com/WordPress/wordpress-develop/pull/7298/commits/0abab555f4918717758351aef54c860a927a5b28